dynatrace/api/openpipeline/settings/processor_endpoint.go

```diff
@@ -69,6 +69,7 @@ type EndpointProcessor struct {
 	fieldsAddProcessor    *FieldsAddProcessor
 	fieldsRemoveProcessor *FieldsRemoveProcessor
 	fieldsRenameProcessor *FieldsRenameProcessor
+	technologyProcessor   *TechnologyProcessor
```

Comment:
nit: should we make this line last in the struct?

Reply:
I added it here so it's aligned with the spec
<img width="448" height="188" alt="image" src="https://github.com/user-attachments/assets/1055b2fb-f36c-40b4-bbe9-19af69ae7e7d" />